### DISCUSSION TITLE
Dat O

### DISCUSSION THREAD

**Discussion by datoishere** [birkenkrahe/ai482#18] [Show and tell]
https://drive.google.com/file/d/10Qa3HbkjQ8HgMCvWqQp6iShm4H2lSlf0/view?usp=sharing

I started recording at 8:00pm and ended at 12:30am. I had to retake this video 9+ times because I kept choking up every time and reshooting a different presentation(a new experience for me). Final video ended up very lengthy so processing took a very long time.

I'm sorry for any mumbling/rambling I did throughout. Did not turn out as expected. Very interesting experience however.

Pledged 12/06/21

**Comment by birkenkrahe** [comment]
Thanks Dat for your effort and comment! - I am sorry you had so much trouble with it, and I'm glad you got something out of the effort. In fact, when I began doing videos (even the simplest ones), years ago, many takes were normal. You have to build stamina against your own mistakes - perfection is the death of the non-professional video maker...nowadays, I usually stick to 1 take and leave all the mumbling and rambling in the video - it actually makes it more worthwhile to listen to! There's something machine-like and artificial about perfection that's off-putting, or put differently, slight imperfections communicate humanity perhaps - as long as the product was created in good faith and with diligence.  I'm beginning to wax philosophically, but now I'm looking forward to watching and hearing your result! Cheers, MB

**Comment by birkenkrahe** [comment]
Just finished watching - and I will watch this again because there was so much interesting information in it. My earlier conjecture holds - I found the "rambling" to actually be effective communication - it's more like a lecture, which is both charming and informative. I found myself to be very inspired by it, especially since I'm going to re-connect with ANNs over the next few weeks. Great work, Dat, enjoy your much-deserved break and I am glad you had fun! This mode of "researching" is very likely what you'll be doing for the rest of your working life if you land an interesting job with some creative challenges. Cheers MB  - PS. I've liked processing (the language behind openprocessing.org) for a long time...might use it in the future after your demo. Lots of non-compsci people use it. D3 is another framework used much lately. Will encounter this in the Data Visualization course.
